_posts/2019-07-10-quarkus-0-19-released.adoc

```diff
@@ -0,0 +1,101 @@
+---
+layout: post
+title: 'Quarkus 0.19.0 released'
+date: 2019-07-10
+tags: quarkus release
+author: gsmet
+---
+
+Quarkus 0.19.0 was just released and is now available on Maven Central.
+
+This new version brings a lot of new extensions, especially in the persistence area.
+
+[NOTE]
+====
+Quarkus 0.19.0 is still targeting GraalVM 19.0.2 as we discovered a bug in GraalVM 19.1.0 that affects Quarkus.
+
+The issue is already fixed but we are waiting for a 19.1.1 release.
+====
+
+== What's new?
+
+Here are the main highlights of this release.
```

Comment:
Here are the main highlights of this release: MongoDB support, Neo4J support, AWS DynamoDB support,  Apache Tika support, Reactive Messaging MQTT support, better Hibernate ORM schema upgrade (useful for dev mode).